spam/spam_tutorial.py

```diff
@@ -722,4 +787,12 @@ def plot_probabilities_histogram(Y):
 # ### Next Steps
 
 # %% [markdown]
-# If you enjoyed this tutorial and you've already checked out the Snorkel 101 Guide, check out the `snorkel-tutorials` table of contents for other tutorials that you may find interesting, including demonstrations of how to use Snorkel for scene-graph detection (images), crowdsourcing, information extraction, data augmentation, and more. (TODO: Many links)
+# If you enjoyed this tutorial and you've already checked out the Snorkel 101 Guide, check out the [`snorkel-tutorials` table of contents](https://github.com/snorkel-team/snorkel-tutorials#snorkel-tutorials) for other tutorials that you may find interesting, including demonstrations of how to use Snorkel
+#
+# * As part of a [hybrid crowdsourcing pipeline](https://github.com/snorkel-team/snorkel-tutorials/tree/master/crowdsourcing)
+# * For [scene-graph detection over images](https://github.com/snorkel-team/snorkel-tutorials/tree/master/scene_graph)
+# * For [information extraction over text](https://github.com/snorkel-team/snorkel-tutorials/tree/master/spouse)
+# * For [data augmentation](https://github.com/snorkel-team/snorkel-tutorials/tree/master/spam)
```

Comment:
This just points back to the spam directory. It might be better to link to the specific notebook, since this directory will have multiple tutorial notebooks.